--- _LaserScan.py	2011-11-06 12:58:53.000000000 +0700
+++ /opt/ros/electric/stacks/common_msgs/sensor_msgs/src/sensor_msgs/msg/_LaserScan.py	2011-11-17 11:35:48.198389504 +0700
@@ -126,7 +126,7 @@
       buff.write(_struct_3I.pack(_x.header.seq, _x.header.stamp.secs, _x.header.stamp.nsecs))
       _x = self.header.frame_id
       length = len(_x)
-      buff.write(struct.pack('<I%ss'%length, length, _x))
+      buff.write(struct.pack('<I%ss'%length, length, _x.encode("utf-8")))
       _x = self
       buff.write(_struct_7f.pack(_x.angle_min, _x.angle_max, _x.angle_increment, _x.time_increment, _x.scan_time, _x.range_min, _x.range_max))
       length = len(self.ranges)
